v1/nix/modules/drv-parts/nodejs-package-json/default.nix

```diff
@@ -38,7 +47,7 @@ in {
           writeScript
           writeScriptBin
           ;
-        inherit (nixpkgs.nodePackages) npm;
+        npm = npm_8 nixpkgs.nodejs;
```

Comment:
If i read this correctly, this means we are vendoring an older npm - which would trigger insecure errors if used from (older) nixpkgs -  and use it with newer nodejs, where it's effectively unsupported?

I understand we need to work around the missing support for v2 and v3 lock files in our builder, but defaulting to the latest supporting nodejs and requiring users to allow the insecure package would be a better solution IMO.

Reply:
solved as discussed